Epam.TestAutomation.Tests/CareersPageTests.cs

```diff
@@ -0,0 +1,41 @@
+using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Web.PageObjects.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Tests;
+
+public class CareersPageTests : BaseTest
+{
+    private CareersPage careersPage;
+
+    [SetUp]
+    public void SetUp()
+    {
+        careersPage = new CareersPage();
+        careersPage.AcceptAllCookies();
+    }
+
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        BrowserFactory.Browser.ClickElement(careersPage.careersButton);
```

Comment:
It is not convenient to use careersPage here. Better to click to the nuttons from the mainPage as you navigate to the MainPage and then after clicking the buttons you should use careersPage.